### PR TITLE
add diagrams

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- add diagrams

## security considerations
None

### CODE REVIEW

File: docs/tenant.puml [hunk 1]
```diff
@@ -0,0 +1,185 @@
+@startuml tenant
+skinparam DefaultTextAlignment center
+skinparam ArrowColor black
```
Comment (pburkholder) [inline docs/tenant.puml:3]: Let's make arrow color blue or purple to better distinguish from other borders.

File: docs/tenant.puml [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+rectangle logsearch {
+node "kibana (N x environment)" as kibana {
+    agent kbn [
+        ==kibana
+        The primary interface for viewing logs
+    ]
+    agent kibanaplugin [
+        ==kibana-auth-plugin
```
Comment (markdboyd) [inline docs/tenant.puml:51]: Is this equal to the https://github.com/cloud-gov/kibana-cf-auth-proxy ?

File: docs/tenant.puml [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+user --> UAAALB
+ALB --> gorouter
+UAAALB --> uaa
+es_k -down-> es_em
+es_i -down-> es_em
+es_m -up-> es_em
+es_em <-down-> es_ed
+kibanaplugin -u-> UAAALB : authenticate users
+kibanaplugin -u-> capi : get user roles
+kibanaplugin -d-> redis : share state\nbetween instances
```
Comment (markdboyd) [inline docs/tenant.puml:171]: If `kibanaplugin == kibana-cf-auth-proxy`, then we're not actually using Redis to share state between instances but cryptographically signed cookies, correct?

Comment (bengerman13) [reply docs/tenant.puml:171]: right - these diagrams are about the current state of the logsearch stack and don't consider the desired state of the opensearch stack

File: docs/tenant.puml (hunk 2 above)
Comment (bengerman13) [reply docs/tenant.puml:51]: no - this is https://github.com/cloud-gov/logsearch-for-cloudfoundry/tree/develop/jobs/kibana-auth-plugin

File: docs/tenant.puml (hunk 1 above)
Comment (bengerman13) [reply docs/tenant.puml]: well, I tried that, but now plantuml decided I get grayscale for every component unless I specify otherwise 🤦‍♂️

Comment (bengerman13) [reply docs/tenant.puml]: alright - everything is recolored. It's ugly, but readable I think

**Review by markdboyd — APPROVED** [review]

**Pull request merged**